packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -165,7 +166,9 @@ const getHoverInputBorderColor = (props: TProps) => {
 const controlStyles = (props: TProps) => (base: TBase, state: TState) => {
   return {
     ...base,
-    fontSize: designTokens.fontSize30,
+    fontSize: props.isCondensed
+      ? designTokens.fontSize20
+      : designTokens.fontSize30,
```

Comment:
I think we should have a token for input when condensed since we are doing this change for multiple input components.
```suggestion
    fontSize: props.isCondensed
      ? designTokens.fontSizeForInputWhenCondensed
      : designTokens.fontSize30,
```
@commercetools/shield-team-design-system what do you think ?